python/version_details/version_details.py

```diff
@@ -586,7 +587,7 @@ def load_data(self, entity):
         self._requested_entity = entity
 
         # If we're pinned, then we don't allow loading new entities.
-        if self._pinned and self.current_entity:
+        if self._pinned and self._current_entity:
```

Comment:
@eliveikis I've done a search thru github repositories for non-dynamite tk components @robblau  gave me access https://github.com/search?p=3&q=org%3Ashotgunsoftware+.current_entity+in%3Afile&type=Code (This search will not work if you don't have access to those repositories) And there was only one other instance where current_entity property was being queried and that seemed safe and should be unaffected by this change.

So I will make these changes now available to @thebeeland  for review.